src/query/interpret/eval.hpp

```diff
@@ -1059,6 +1063,7 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
   FrameChangeCollector *frame_change_collector_;
   /// Property lookup cache ({symbol: {property_id: property_value, ...}, ...})
   mutable std::unordered_map<int32_t, std::map<storage::PropertyId, storage::PropertyValue>> property_lookup_cache_{};
+  const int64_t *hops_counter_;
```

Comment:
Pointer to an `int` looks suspicious 🤔 Is it possible to access the value via `EvaluationContext` (there is the `ctx_` ptr)? 👀

Reply:
EvaluationContext does not have a hops counter though. That's in the execution context